package.json

```diff
@@ -9,12 +9,12 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon -e ts --exec 'ts-node src/index.ts'",
+    "start": "nodemon -e ts --exec ts-node src/index.ts",
     "debug": "ts-node --inspect --compilerOptions '{\"inlineSources\":true}' src/index.ts",
-    "build:editor": "cd src/editor && yarn && npm run build && cd -",
+    "build:editor": "cd src/editor && yarn && npm run build && cd .. && cd ..",
     "build:typescript": "tsc",
     "copy:graphql": "cp src/*.graphql dist/",
-    "copy:editor": "mkdir -p dist/editor && cp src/editor/*.{html,js,css,svg} dist/editor",
+    "copy:editor": "(mkdir -p dist/editor || mkdir dist\\editor || true) && cp src/editor/*.{html,js,css,svg} dist/editor",
```

Comment:
> btw should it care at all, e.g. copying graphql does not

@mac2000 you can move `mkdir dist` into `build:all` and have `mkdir dist/editor` in `copy:editor`

Reply:
![image](https://user-images.githubusercontent.com/88868/81305335-e4640900-9086-11ea-8d09-817af571b98b.png)

unfortunatelly concrete this one is not working :(

Reply:
@mac2000 Can you try `mkdir "a/b"`?

Reply:
Did worked out provided suggested changes not it even better then was, thank you!